clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -745,7 +745,9 @@ LValue CIRGenFunction::buildDeclRefLValue(const DeclRefExpr *E) {
       if (auto *FD = LambdaCaptureFields.lookup(VD))
         return buildCapturedFieldLValue(*this, FD, CXXABIThisValue);
       assert(!UnimplementedFeature::CGCapturedStmtInfo() && "NYI");
-      llvm_unreachable("NYI");
+      // TODO[OpenMP]: Find the appropiate captured variable value and return
+      // it.
+      // TODO[OpenMP]: Set non-temporal information in the captured LVal.
```

Comment:
Same for this and all of the TODOs that make sense doing so.

Reply:
You haven't address this one. Please add two `assert(!UnimplementedFeature::....` here in place of these TODOs.

Reply:
Done.